docs/track.rst

```diff
@@ -1893,7 +1895,7 @@ If you want it to delete one specific index template instead, you can specify th
 
 * ``template`` (mandatory): The name of the composable template that should be deleted.
 * ``only-if-exists`` (optional, defaults to ``true``): Defines whether the index template should only be deleted if it exists.
-* ``delete-matching-indices`` (optional, defaults to ``false``): Whether to delete indices that match the index template's index pattern.
+* ``delete-matching-indices`` (optional, defaults to ``false``): Whether to delete indices that match the index template's index pattern. This setting is ignored in `Elastic Serverless <https://docs.elastic.co/serverless>`_ - please use data streams and ``delete-data-stream`` operation instead.
 * ``index-pattern`` (mandatory iff ``delete-matching-indices`` is ``true``): Specifies the index pattern to delete.
```

Comment:
Noticed while passing through:
```suggestion
* ``index-pattern`` (mandatory if ``delete-matching-indices`` is ``true``): Specifies the index pattern to delete.
```

Reply:
The "iff" is an acronym for "if and only if". It's present in a couple of places in Rally code and docs. I'll remove its occurrences in docs. The first time I saw it I also thought it's a typo.